src/engraving/dom/system.cpp

```diff
@@ -664,6 +664,10 @@ void System::scanElements(void* data, void (* func)(void*, EngravingItem*), bool
         }
     }
     for (SpannerSegment* ss : m_spannerSegments) {
+        if (!ss->spanner()) {
```

Comment:
spanner() should never be null for a SpannerSegment, so I think this check is just covering some deeper problem